boilerplate/app/app.tsx

```diff
@@ -98,11 +99,13 @@ export function App() {
         {/* @demo remove-block-start */}
         <AuthProvider>
           {/* @demo remove-block-end */}
-          <AppNavigator
-            linking={linking}
-            initialState={initialNavigationState}
-            onStateChange={onNavigationStateChange}
-          />
+          <ThemeProvider>
```

Comment:
Like this change, when we were reviewing the ErroBoundary stuff I was thinking it was too low being in the AppNavigator